front-end/src/app/app.component.html

```diff
@@ -2,8 +2,12 @@
   <div class="menu"></div>
   <div class="home-page">
     <div class="home-header">
-      <h1 class="title">{{ title }}</h1>
-      <h2 class="subtitle">{{ nameOfRoom }}</h2>
+      <div class="title-box">
+        <h1 class="title">{{ title}}</h1>
```

Comment:
Needs space after title, probably better to run prettier since it will fix all of these styling issues

Reply:
can you run prettier after i fix the comments?

Reply:
sure